src/api/public/apidocs-new/components/parameters/repository_name.yaml

```diff
@@ -3,5 +3,5 @@ name: repository_name
 schema:
   type: string
 required: true
-description: Repository name
+description: "Repository name"
```

Comment:
Is this change really needed? Should we add `""` to all the `*_name* parameters?

Reply:
I'm affraid so. I'm preparing another PR that fixes several of those issues with our documented endpoints after passing some linter against them. If you want to check it by yourself: https://github.com/stoplightio/spectral